bin/ch-test

```diff
@@ -111,12 +111,20 @@ builder_set () {
 dir_img_mk () {
     dir_img_rm
     printf "creating %s\n\n" "$CH_TEST_IMGDIR"
-    mkdir "$CH_TEST_IMGDIR"
+    if [[ -n $SLURM_JOBID ]]; then
+        srun --ntasks-per-node=1 mkdir "$CH_TEST_IMGDIR"
+    else
+        mkdir "$CH_TEST_IMGDIR"
+    fi
```

Comment:
I think this if statement can be replaced with `$ch_mpirun_node`.

Reply:
`$ch_mpirun_node` is undefined here at the ch-test execution. I suppose we could source common.bash like the bat files; thoughts?

Reply:
it appears it could be used multiple times. I think sourcing it would be helpful.